### PR TITLE
adding redisgraph .NET tutorial

### PR DESCRIPTION
**Pull request by slorello89**
Redis Graph tutorial for .NET users

### CODE REVIEW

File: docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+```bash
+docker run -p 6379:6379 redislabs/redisgraph
+```
+
+The above command will start an instance of Redis locally with the RedisGraph module loaded, and you will be able to connect to it on **localhost:6379**
+
+## Connecting
+
+NRedisGraph makes use of the [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis) project which is installed along with `NRedisGraph` to create the `RedisGraph` object you'll first create a `ConnectionMultiplexer`, and pull a reference, pull a reference to an `IDatabase` object from it, and then initialize the `RedisGraph` with the `IDatabase` object:
```
Comment (simonprickett) [inline docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md]:
```suggestion
NRedisGraph makes use of the [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis) project which is installed along with `NRedisGraph`. To create the `RedisGraph` object you'll first create a `ConnectionMultiplexer`, and pull a reference to an `IDatabase` object from it, and then initialize the `RedisGraph` with the `IDatabase` object:
```

File: docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+```csharp
+var muxer = ConnectionMultiplexer.Connect("localhost");
+var db = muxer.GetDatabase();
+var graph = new RedisGraph(db);
+```
+
+## Sending a Query
+
+Querying in Redis Graph applies to a wide array of operations, but fundamentally when executing queries with NRedisGraph, all you need to do is execute `graph.Query` or `graph.QueryAsync` passing in the name of the graph you want to query and the query you want to run.
```
Comment (simonprickett) [inline docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md]:
```suggestion
Querying in RedisGraph applies to a wide array of operations, but fundamentally when executing queries with NRedisGraph, all you need to do is execute `graph.Query` or `graph.QueryAsync` passing in the name of the graph you want to query and the query you want to run.
```

File: docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+### Creating a Node
+
+To create a node in RedisGraph, you'll use the Create Operation. Let's start by making 2 Humans, Alice and Bob:
```
Comment (simonprickett) [inline docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md:41]: Might be good to talk about the graph being stored at a Redis key `pets` before showing code that uses that key?

File: docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+### Creating Relationships
+
+Like creating nodes, you can also create relationships in RedisGraph using the `Query`/`QueryAsync` commands to create relationships between nodes in RedisGraph. For example, to establish the owner relationship between Bob and the Greyhound Honey, you would use the following:
+
+```csharp
+await graph.QueryAsync("pets",
+    "MATCH(a:human),(p:pet) WHERE(a.name='Bob' and p.name='Honey') CREATE (a)-[:OWNS]->(p)");
+```
+
+You could establish other relationships as well between nodes, say, for example, both Bob and Alice both walk honey you could add the connections:
```
Comment (simonprickett) [inline docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md]:
```suggestion
You could establish other relationships as well between nodes, say, for example, both Bob and Alice both walk Honey you could add the connections:
```

File: docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+```csharp
+await graph.QueryAsync("pets",
+    "MATCH(a:human),(p:pet) WHERE(a.name='Alice' and p.name='Honey') CREATE (a)-[:WALKS]->(p)");
+await graph.QueryAsync("pets",
+    "MATCH(a:human),(p:pet) WHERE(a.name='Bob' and p.name='Honey') CREATE (a)-[:WALKS]->(p)");
+```
+
+### Querying Relationships
+
+Now that we've created a few Nodes and Relationships between nodes, we can query things in the Graph, again using the `Query` and `QueryAsync`. So, for example, if we wanted to find all of Honey's owners, we would issue the following query:
```
Comment (simonprickett) [inline docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md]:
```suggestion
Now that we've created a few Nodes and Relationships between nodes, we can query things in the Graph, again using `Query` and `QueryAsync`. So, for example, if we wanted to find all of Honey's owners, we would issue the following query:
```

File: docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+}
+```
+
+We can also find all the walkers of Honey by finding all the human's who have a `WALKS` relationship with Honey:
+
+```csharp
+matches = await graph.QueryAsync("pets", "MATCH(a:human),(p:pet) where (a)-[:WALKS]->(p) and p.name='Honey' return a");
+```
+
+Then if we wanted to find all of Bob's dogs, we would query the graph and find all the canine's who have an OWNS relationship with a human named Bob:
```
Comment (simonprickett) [inline docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md]:
```suggestion
Then if we wanted to find all of Bob's dogs, we would query the graph and find all the canines who have an OWNS relationship with a human named Bob:
```

**Review by simonprickett — APPROVED** [review]
Looks good and works on my local build... I left some minor suggested commits.

File: docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md (hunk 3 above)
Comment (slorello89) [reply docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md:41]: Added some text above about how pets is the key to use for the graph.

**Pull request merged**